api/app.py

```diff
@@ -38,10 +49,27 @@ async def lifespan(app: FastAPI):
     for job in app.schedule.get_jobs():
         ap_logger.info({"Name": str(job.id), "Run Frequency": str(job.trigger), "Next Run": str(job.next_run_time)})
 
+
+    database = {
+        "host": config("pg_host", default="localhost"),
+        "dbname": config("pg_dbname", default="orpy"),
+        "user": config("pg_user", default="orpy"),
+        "password": config("pg_password", default="orpy"),
+        "port": config("pg_port", cast=int, default=5432),
+        "application_name": config("APP_NAME", default="PY"),
+    }
+
+    database = " ".join("{}={}".format(k, v) for k, v in database.items())
```

Comment:
Is it possible to pass arguments to the connexion constructor instead of a string, this would help us later to provide dynamic arguments

Reply:
Yes, using `kwargs`

Reply:
Done in https://github.com/openreplay/openreplay/pull/1703/commits/24713b0691c27763a6550d599bbefa60f4303dc1